source/reference/cutover-process.txt

```diff
@@ -76,6 +114,21 @@ Steps
       ``true``. If ``canWrite`` is ``false``, wait until ``progress``
       shows ``canWrite`` is ``true``.
 
+      Request
+      ~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/progress.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/progress.json
+         :language: json
+         :emphasize-lines: 5
```

Comment:
[Question]

Is this highlight on the correct line? It's currently on `"canCommit":true,`, but the introduction to thise code block talks about `canWrite`. Do we maybe want this highlight on line 6 instead?

Reply:
I realized I made this mistake because the example response I should use needs to have canWrite as true, but in this example canWrite is false. Fixing!